client/src/app/pages/vulnerability-list/vulnerability-context.tsx

```diff
@@ -44,12 +44,11 @@ export const VulnerabilitySearchProvider: React.FunctionComponent<
     tableName: "vulnerability",
     persistenceKeyPrefix: TablePersistenceKeyPrefixes.vulnerabilities,
     columnNames: {
-      identifier: "Name",
+      identifier: "ID",
       title: "Description",
       severity: "CVSS",
-      published: "Published",
-      packages: "Packages",
-      sboms: "SBOMs",
+      published: "Date published",
+      sboms: "Related documents",
```

Comment:
I am not sure what we will do with this column, we might need to ask PM or give it a though ourselves. But that can be done in a separate PR